wg-notebooks/charter.md

```diff
@@ -0,0 +1,39 @@
+# WG Notebooks Charter
+
+This charter adheres to the conventions, roles and organization management outlined in [wg-governance].
+
+## Scope
+
+WG Notebooks is responsible for the user experience around Notebooks and other IDEs and their integrations with Kubeflow.
```

Comment:
It is confusing that WG Notebooks would be responsible for "other IDEs and their integrations" - wouldn't it make sense to name the WG something more general then? e.g. WG IDEs, WG Developers, WG Development

Reply:
I think `WG Notebooks` is acceptable, the issue with `WG Developers` is that its even more likely to be confused with Kubeflow devs, whereas `WG IDE` is more correct, but less understandable.

Reply:
This does seem a bit over-broad/general as a scope -- we are erring on the side of narrowly defined scopes and/or limiting to existing functionality in our working groups for now to ensure that there are very clearly defined areas of responsibility.

Reply:
@theadactyl literally I cant see what else a "Notebook" working group could be responsible for.

Can you clarify what is too general?

Reply:
I think @theadactyl was referring to including other IDEs in scope.